src/components/GoalItem.tsx

```diff
@@ -0,0 +1,121 @@
+import { Text } from '@geist-ui/core';
+import styled from 'styled-components';
+import { useIntl, useNow, useTranslations } from 'next-intl';
+
+import type { Goal, Scalars, Tag, UserAnyKind } from '../../graphql/@generated/genql';
+import { accentIconColor, goalBackgroundHover } from '../design/@generated/themes';
+
+import { Tag as TagItem } from './Tag';
+import { Icon } from './Icon';
+import { UserPic } from './UserPic';
+
+interface GoalItemProps {
+    title: string;
+    issuer?: UserAnyKind;
+    id: string;
+    tags?: Tag[];
+    createdAt: Scalars['DateTime'];
+    owner?: UserAnyKind;
+    comments?: number;
+    hasForks?: boolean;
+}
+
+/* <GoalItem
+    id={goal.id}
+    title={goal.title}
+    projectTitle={goal.project?.title}
+    tags={goal.tags}
+    issuer={goal.computedIssuer}
+    createdAt={goal.createdAt}
+    updatedAt={goal.updatedAt}
+/> */
+
+const StyledGoal = styled.div`
+    display: flex;
+    align-items: center;
+    &:hover {
+        background-color: ${goalBackgroundHover};
+    }
+    border-top-right-radius: 6px;
+    border-bottom-right-radius: 6px;
+
+    padding: 10px 24px;
+`;
+
+const StyledStatus = styled.div`
+    width: 30px;
+`;
+
+const StyledName = styled.div`
+    min-width: 615px;
+`;
+
+const StyledTitle = styled.div``;
+
+const StyledAddon = styled.div`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding-left: 25px;
+    padding-right: 25px;
+`;
+
+const StyledCommentsCount = styled.span`
+    font-size: 14px;
+    margin: 0 9px;
+`;
+
+export const GoalItem: React.FC<GoalItemProps> = ({
+    id,
+    owner,
+    issuer,
+    createdAt,
+    tags,
+    title,
+    comments,
+    hasForks,
+}) => {
+    const t = useTranslations('goals.item');
+    const intl = useIntl();
+    const now = useNow({
+        updateInterval: 1000 * 10,
+    });
+
+    return (
+        <StyledGoal>
+            <StyledStatus>
+                <Icon size="s" type="exclamationCircle" color={accentIconColor} />
+            </StyledStatus>
+            <StyledName>
+                <StyledTitle>
+                    <Text font={'16px'} b marginRight="16px">
+                        {title}
+                    </Text>
+                    {tags?.map(
+                        (tag) =>
+                            tag && (
+                                <TagItem
+                                    key={tag.id}
+                                    title={tag.title}
+                                    description={tag.description}
+                                    color={tag.color}
+                                />
+                            ),
+                    )}
+                </StyledTitle>
+                <Text font={'12px'} span>
+                    {`${id} ${t('Created')} ${intl.formatRelativeTime(createdAt, now)} ${t('by')} ${issuer?.name}`}
```

Comment:
I think you don't need `now` via hook here. We will have live updates for pages. It means all data will be updated automatically.

Reply:
done